src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -679,14 +679,11 @@ Json::Value checkFee (
         feeTrack.scaleFeeLoad (feeDefault,
             ledger->fees().base, ledger->fees().units, isUnlimited (role));
     std::uint64_t fee = loadFee;
-    if (ledger->rules().enabled(featureFeeEscalation,
-        config.features))
```

Comment:
No, and I should have added a comment here preemptively. The reason I removed the check here is that `getMetrics()` does the check as well, and returns `boost::none` if the feature is not enabled. I figured the overhead of checking twice (particularly now that it is enabled on the network), isn't worth the overhead of processing the `x-assume-tx` parameter, and so should be marginally more efficient, and less repetitive. Related, that's why I removed the `assert`, too - it could come back uninitialized, and in fact, does in several unit tests.